lib/connect-backbone-to-react.js

```diff
@@ -93,6 +93,14 @@ module.exports = function connectBackboneToReact(
       }
 
       createNewProps() {
+        // Bail out if our component has been unmounted.
+        // The only case where this flag is encountered is when this component
+        // is unmounted within an event handler but the 'all' event is still triggered.
+        // It is covered in a test case.
+        if (this.hasBeenUnmounted) {
```

Comment:
Though one thing of note is that devs are more willing to call methods in tests when not underscored. I'd like to ensure that calling public methods of React Components in tests should be used sparingly.